mnefun/_sss.py

```diff
@@ -577,16 +577,22 @@ def _pos_valid(pos, dist_limit, gof_limit):
 def _get_t_window(p, raw):
     t_window = p.coil_t_window if p is not None else 'auto'
     if t_window == 'auto':
-        from mne.chpi import _get_hpi_info
         try:
             hpi_freqs, _, _ = _get_hpi_info(raw.info, verbose=False)
+            info = raw.info
+            line_freq = info['line_freq'] or 60
+            line_freqs = np.arange(line_freq, info['sfreq'] / 3., line_freq)
```

Comment:
In general you want as few as possible commands in the `try` part as possible. In this case the `try` has to do with `_get_hpi_info` failing on old files IIRC, so your new lines of code should all live in the `else` clause